### PR TITLE
Add Rancher Cluster Operations Module

### PR DESCRIPTION
**Pull request by git-ival**
Adds a utility/helper root module to setup components needed for various rancher features

Currently supports setting up Rancher to enable a controller metrics Grafana dashboard. Assumes Rancher server has `CATTLE_PROMETHEUS_METRICS` set to true.

### CODE REVIEW

File: rancher-cluster-operations/modules/rancher-controller-metrics/files/controllers-dashboard.json [hunk 1]
```diff
@@ -0,0 +1,797 @@
...
+      },
+      "type": "graph",
+      "xaxis": {
+        "buckets": null,
+        "mode": "time",
+        "name": null,
+        "show": true,
+        "values": []
+      },
+      "yaxes": [
```
Comment (rmweir) [inline rancher-cluster-operations/modules/rancher-controller-metrics/files/controllers-dashboard.json:275]: Is "yaxes" valid?

**Review by rmweir — CHANGES_REQUESTED** [review]

File: rancher-cluster-operations/modules/rancher-controller-metrics/files/controllers-dashboard.json (hunk 1 above)
Comment (git-ival) [reply rancher-cluster-operations/modules/rancher-controller-metrics/files/controllers-dashboard.json:275]: Yes, this was generated by exporting the dashboard from grafana. Apparently "axes" is the plural for "axis", in this case its defining the number of metrics that need a y-axis for this graph.

Comment (rmweir) [reply rancher-cluster-operations/modules/rancher-controller-metrics/files/controllers-dashboard.json:275]: Ah okay, was just wondering why there wouldn't be xaxes as well.

**Review by rmweir — APPROVED** [review]
LGTM, just skimmed and checked it as well as I could with limited information.

**Pull request merged**